e2e/e2e_test.go

```diff
@@ -225,18 +225,23 @@ var _ = Describe("e2e", func() {
 			err = k8sClient.Update(context.Background(), nhc)
 			Expect(err).ToNot(HaveOccurred(), "minHealthy update should be allowed")
 
-			By("waiting for reboot")
-			Eventually(func() (time.Time, error) {
-				bootTime, err := utils.GetBootTime(clientSet, nodeUnderTest.Name, testNsName, log)
-				if bootTime != nil && err == nil {
-					log.Info("got boot time", "time", *bootTime)
-					return *bootTime, nil
-				}
-				log.Error(err, "failed to get boot time")
-				return time.Time{}, err
-			}, nodeRebootedTimeout, 30*time.Second).Should(
-				BeTemporally(">", testStart),
-			)
+			// TODO node reboot doesn't work on k8s
+			// investigate if this is something we can fix in SNR
+			// not relevant for NHC itself, so ignore for now
+			if _, exists := os.LookupEnv("SKIP_FOR_K8S"); !exists {
```

Comment:
Nit: perhaps add a log or a step noting that this validation is skipped

Reply:
good idea, done